src/sass/patterns/_data-tables.scss

```diff
@@ -19,3 +19,21 @@
     padding-right: 0;
   }
 }
+
+// Override LDS table cell padding to make click target fill the whole row
+.table-row-targets {
+  label {
+    cursor: pointer;
+  }
+
+  .slds-checkbox__label {
+    padding: $spacing-medium $spacing-large;
+  }
+
+  &.slds-table_cell-buffer tr > th:first-child,
+  &.slds-table_cell-buffer tr > td:first-child,
+  &.slds-table th:first-child,
+  &.slds-table td:first-child {
```

Comment:
As far as I can tell `.slds-table_cell-buffer` is only applied to `.slds-table`, so I don't see how these first two definitions could differ from the last two. Can we just remove the `.slds-table_cell-buffer` ones?